release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -993,21 +1092,190 @@ If you are using the option `--keep-manifest-list=true`, the only valid value fo
+
+* Previously, the bootstrap rendering logic failed to detect a usable machine network CIDR when using IPv6 dual stack mode unless the IPv4 CIDR was the first element in the install-config machine network CIDR array. The parsing logic was fixed to loop through all machine network CIDRs, so the IPv4 address is now correctly loaded among the machine network CIDRs in dual stack mode. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1907872[*BZ#1907872*])
+
+* Previously, if the `openshift-etcd` namespace was deleted, the `etcd-endpoints` config map was not re-created, and the cluster would not recover. The `etcd-endpoints` config map is now re-created if it is missing, allowing the cluster to recover. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1916853[*BZ#1916853*])
+
+*Image Registry*
+
+* The last Kubernetes update enforced a timeout on APIs. This timeout results in every long standing request being dropped after 34 seconds. When importing large repositories, specifically ones with several tags, the timeout is reached, not allowing the import to succeed as in previous versions. There is a flag to set a different timeout on `oc` client but there was not an example provided, making it difficult for the client to understand how to bypass the API timeout. Providing an example of the flag usage on `oc` help made things clear for the client, now it is easier to find this option. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1878022[*BZ#1878022*])
+
+* Previously, using two distinct versions of the same logging package resulted in Operator logs being partially lost. This fix makes logging package versions equal, which means the upgraded logging package used by the Operator matches the one used by client-go. Now logs are not lost. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1883502[*BZ#1883502*])
+
+* Previously the pruner was trying to detect the registry name using image streams, but when there were no image streams the pruner failed to detect the registry name. With this fix, the Image Registry Operator provides the pruner with the registry name. Now the pruner does not depend on the existence of image streams to detect the registry name. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1887010[*BZ#1887010*])
+
+* Previously the Operator pod did not have memory requests, which in case of memory pressure on the node, the Operator could be OOM-killed before other BestEffort containers. This fix added memory requests. Now the Operator is not OOM-killed if there are other BestEffort containers on the node. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1888118[*BZ#1888118*])
```

Comment:
Unclear jargon here (OOM-killed, BestEffort containers)

Reply:
spelling out "out-of-memory". BestEffort is a Kubernetes term apparently. This should be in backticks though, so I'll update that.